server.js

```diff
@@ -18,11 +41,37 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+app.get('/', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  res.json(thoughts)
 })
 
+app.post('/', async (req, res) => {
+  const { message } = req.body
+  const thought = new Thought({
+    message: message
+  })
+
+  try {
+    const savedThought = await thought.save()
+    res.status(201).json(savedThought)
+  } catch (err) {
+    res.status(400).json({ message: 'Could not save happy thought to database', error: err.errors })
+  }
+})
```

Comment:
You created the `Name` field in your model, and in the form, but you do not use it here. Add it, please :)

Reply:
I got some help from Van to get the anonymous posts to work in my frontend. The solution was to remove the name from this part of the code. So I am using it, I think?